elastic-agent-client.proto

```diff
@@ -25,6 +41,8 @@ message ConnInfo {
     bytes peer_cert = 5;
     // Peer private key.
     bytes peer_key = 6;
+    // Allowed services that spawned process can use. (only used in V2)
+    repeated ConnInfoServices services = 7;
```

Comment:
What is it expected that processes do with the services list? Is the intent that each process checks to make sure the services it needs are actually available?

Reply:
Correct. A process can only use the services that it is given, if it tries to Elastic Agent will return an error back to the caller.